### ISSUE TITLE
Reincarnation

### ISSUE THREAD

**Issue by shyzus** [shyzus/Legacy#2]
# Reincarnation for players that have died
First things first, reincarnation replaces the ability to remake. Remaking while alive will allow you to choose you're preferred race. Reincarnation does not. If you die you have two options: roleplay or reincarnate.

## How
Once you die some time will pass and you will be given the option reincarnate trough a prompt. If you agree you will be reincarnate into a random race that is not the same race that you are now, on a random planet and start off as a first generation character just like anyone else. Any previous characters are considered lost. The character you were on when you reincarnated will be transformed into a cloud npc and join the other cloud npc's for a certain amount of time before being deleted to prevent clutter. While cloud npc's are present they will randomly say things in character. Things that you the original character have said before in your life time and can be found in your log files.

If you deny, you will stay as you are in the afterlife and roleplay as you please. Which is preferred. Reincarnation is a last resort if you no longer wish to roleplay in the afterlife and have given up on any joy from your current character and family line and want to start over.

## This is an idea
Feel free to suggest changes below or ask questions regarding this mechanic.

**Comment by Matokushin** [comment]
To give people a positive outlook on reincarnating, let's look at Uub. Starting out he didn't seem strong or wasn't/didn't know how to use his power or unlock his potential from being reincarnated from Kid Buu. We all know kid buu was literally about to end dbz before they got saved at that last minute because he was naturally op. Okay so Uub meeting Goku, Goku sensed that he was strong or whatever and after Goku helped him unlock said power it became to slowly show overtime.

Basically, upon reincarnating 65% of your gains should be invisible, like wielding a sword in the new sword system, and you'll have to train it back up. (Optional Sidenote: If a person gains lower than the server average gain after year fifteen they get server average gain +5 or however you want to modify their small gain so they won't be below server average.)

**Comment by shyzus** [comment]
I think for the time being so legacy v1. No reincarnation process you only remake to 1st gen both living and dead. revival is there and il simply work on giving witches/wizards a way to revive people as another option. So total options to comeback from the death: Wishes, Kai revive, Demon revive and Wizard/Witch ritual.

**Comment by Trazz666** [comment]
So, this will be my final thoughts on the death thing as of now. Death sucks, and that phrase should reflect every aspect of the death process. Firstly, I feel all manner of revival should be limited. Kai/Demon revive and DB wishes are pretty self explanatory. Witch/Wizard revival should have some negative effect, such as witches require someone to sacrifice themselves to bring back the revived, and wizards enslave those who are brought back (either against their will or not) along with the large magic/energy drain. Techies reviving someone will require a butt load of materials, naturally, either from the tech's stores or from the player paying beforehand. 
As for reincarnation, I believe that should only be implemented for someone trying to remake after death. I feel after a grace period of 1 or 2 IRL weeks, they should be allowed to 'fade' and remake into what they want. That gives them enough time to make a legitimate effort into revival and RP, but also gives them a safe out for their patience. For those impatient and reincarnates before the time period, they should receive some kind of penalty, like a lasting negative mutation or deformity. 

And one last thought. Bring ripped back into the mortal plane, no matter the means, should be accompanied by a brief period of debilitation. Your spirit should take a period of adjustment time in it's new shell to become fully powered. Perhaps anywhere between 1 day and 1 week, I'd say.

**Comment by Matokushin** [comment]
I agree with both of your points. I personally, don't care about reincarnation really I was just giving some more insight on how it probably should go.  I second Trazz's point when he said your body should have a period of adjustment time. That's just like when Ginyu body swaps he has to get adjusted to the new body before he can efficiently use it. Great idea.

**Comment by shyzus** [comment]
Will take this idea into consideration wether to implement this at v1 or after, so definitly a form of reincarnation will be implemented.
